models/models.js

```diff
@@ -1,8 +1,35 @@
 const db = require("../db/connection")
+const fs = require("fs/promises")
+const path = require("path");
+
 
 
 exports.selectTopics = () => {
     return db.query('SELECT * FROM topics;').then((topics) => {
         return topics.rows
     })
-}
\ No newline at end of file
+}
+
+exports.selectEndpoints = () => {
+    const filePath = path.resolve(__dirname, '../endpoints.json')
+    return fs.readFile(filePath, 'utf-8')
+     .then((data) => {
+        return data
+        
+    }).catch((err) => {
+        return err
+    })
```

Comment:
This is not wrong, but we don't have to always use promises. As you've got Endpoint.json in the repo, could simply require it in. When you require it, it'll be imported as an object
Also you may want to chain the .catch in the controller rather than here in the model